fs/fs_zenfs.cc

```diff
@@ -1016,8 +1155,258 @@ Status ZenFS::Mount(bool readonly) {
     files_mtx_.unlock();
   }
 
-  Info(logger_, "Superblock sequence %d", (int)superblock_->GetSeq());
-  Info(logger_, "Finish threshold %u", superblock_->GetFinishTreshold());
+  Info(logger_, "Superblock sequence %d", (int)metadata_superblock_->GetSeq());
+  Info(logger_, "Finish threshold %u", metadata_superblock_->GetFinishTreshold());
+  Info(logger_, "Filesystem mount OK");
+
+  if (!readonly) {
+    Info(logger_, "Resetting unused IO Zones..");
+    zbd_->ResetUnusedIOZones();
+    Info(logger_, "  Done");
+  }
+
+  LogFiles();
+
+  return Status::OK();
+}
+
+/* Mount the filesystem by recovering form the latest valid snapshot zone
+ * and metadata zone*/
+Status ZenFS::MountV2(bool readonly) {
```

Comment:
Same as `MkfsV2`, please reuse original function (you can take some logic out and create new function to handle the differences)